### ISSUE TITLE
Misleading/incorrect statement in Opus codec registration

### ISSUE THREAD

**Issue by Vanilagy** [w3c/webcodecs#853]
The Opus codec registration states the following:
![image](https://github.com/user-attachments/assets/03f43b70-5563-464a-b060-03ea6441e877)

The conclusion I draw from this is that when `description` is set, the bitstream is in the 'ogg' format. However, this is not necessarily the case. `description` may still be provided even if the format is 'opus', and the 'ogg' format may not even be supported even if description is being set.

Here's a repro link: https://jsfiddle.net/7d1fLmbz/4/
I'm on Chrome Version 131.0.6778.86 on Windows 11.

I think what the registration is meaning to say is that each user agent is free to set `description` if they want, it's just when they set it it needs to be an Identification Header. It's very useful that it does this and I make use of this `description` field myself. However, the presence of  `description` cannot be used to conclude whether the format is 'opus' or 'ogg'.

**Comment by padenot** [comment]
Yes this needs a bit of work, I don't think this was ever implemented by any browser, per spec, we might want to change the spec.
